test/ponyfill/regexp-compat.test.ts

```diff
@@ -127,6 +127,11 @@ const testCases: TestCase[] = [
     flags: 'y',
     strings: ['a', 'bb', 'bab'],
   },
+  {
+    source: '(a(b))',
+    flags: '',
+    strings: ['abc'],
```

Comment:
I'm not sure this additional test case is correct.

I know currently the test implementation does not have a feature to check captures.
But, I think you can such a case correctly by using back-reference, e.g source `(a(b))\1` and string `abab`.

Reply:
> I know currently the test implementation does not have a feature to check captures.

https://github.com/MakeNowJust/rerejs/blob/272bcaa956c9d2ff1d4095b5e37f85588b69058e/test/ponyfill/regexp-compat.test.ts#L145

I think this code is check captures...

Reply:
Okay. It seems good.